components/tag-list/tag-list.js

```diff
@@ -24,12 +40,20 @@ class TagList extends ArrowKeysMixin(LitElement) {
 			.tag-list-container {
 				display: flex;
 				flex-wrap: wrap;
-				margin: -6px -6px 0 0;
+				margin: -${MARGIN_TOP_HEIGHT}px -6px 0 0;
```

Comment:
Haha I had this before too!  See thread here: https://github.com/BrightspaceUI/core/pull/2236#discussion_r830198507
It seems to work because it's an integer, but maybe we're good with that and if it ever breaks visual-diff _should_ catch it

Reply:
Haha interesting! I can change back to hard coded then or we can keep an eye on it